### ISSUE TITLE
iOS: Accessibility text for "Action" button should be "Share"

### ISSUE THREAD

**Issue by vincode-io** [Ranchero-Software/NetNewsWire#1582]
When viewing an article, I would suggest that the VoiceOver label for the “Action” button be changed to “Share”.

This is minor and probably nit-picky, but would bring this inline with how this button is typically labeled system-wide and in most apps.
